topology/nhlt/nhlt-processor.c

```diff
@@ -96,7 +96,7 @@ static int print_as_hex_bytes(uint8_t *manifest_buffer, uint32_t manifest_size,
 	char *dst;
 	int i;
 
-	bytes_string_buffer = calloc((manifest_size + nhlt_size) * ALSA_BYTE_CHARS,
+	bytes_string_buffer = calloc((manifest_size + nhlt_size + 1) * ALSA_BYTE_CHARS,
```

Comment:
Lets call this out in a comment if the NHLT or the manifest user a NULL termination and if/why/where we need to add these in the code.

Reply:
Ack, updated the commit message and the patch, add 1 byte instead of ALSA_BYTE_CHARS.